docs/cli/app-types.md

```diff
@@ -0,0 +1,141 @@
+# .NET Core CLI types of portability of applications
+
+## Introduction
+There are several ways to think about the "types" of applications you can build. Usually, the types describe 
```

Comment:
Do we need the quotes here on this paragraph? I'll refrain myself from other stylistic suggestions :smile: